test/e2e-wi-tests.sh

```diff
@@ -157,7 +157,7 @@ function gcp_auth_setup() {
 
 # Create a cluster with Workload Identity enabled.
 # We could specify --cluster-version to force the cluster using a particular GKE version.
-initialize $@ --cluster-creation-flag "--workload-pool=\${PROJECT}.svc.id.goog"
+initialize $@ --kubetest2-flag "--workload-pool=\${PROJECT}.svc.id.goog"
```

Comment:
This should now be
```suggestion
initialize $@ --kubetest2-flag "--enable-workload-identity=true"
```

Reply:
Done. And thank you for the fix upstream!